src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,45 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
+        CinemaHall kyiv = new CinemaHall();
+        kyiv.setCapacity(100);
+        kyiv.setDescription("Kyiv cinema-hall");
+        cinemaHallService.add(kyiv);
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(movieService.get(fastAndFurious.getId()));
+        movieSession.setShowTime(LocalDateTime.parse("2007-12-03T10:15:30"));
```

Comment:
Using `LocalDateTime.parse` assumes that the input string is in the correct ISO-8601 format. If there's any chance that the format might differ, consider using a `DateTimeFormatter` to specify the expected format explicitly.